tutorials/pages/send-shimmer-tokens-with-iota-sdk.md

```diff
@@ -0,0 +1,148 @@
+
+Here again we read the environment variables from the `.env` file and then we recreate the account manager which will use the Stronghold file and database that were created in the previous step.
+
+```js
+// To create an address we need to unlock stronghold.
+await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
+
+const address = (await account.generateEd25519Addresses(1))[0];
+
+console.log(`Generated address:`, address.address);
+```
+
+You can see all testnet addresses begin with `rms`, which is the reverse of what real Shimmer addresses start with. This is how you can tell testnet and real addresses apart. You can get some testnet tokens from the [faucet](https://faucet.testnet.shimmer.network).
+
+## Check the balance
+
+Now you should have some tokens. To validate that, we can use the library to inspect our account.
+
+```js
+// Sync new outputs from the node.
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const _syncBalance = await account.sync();
+
+// After syncing the balance can also be computed with the local data
+const balance = await account.getBalance();
+console.log('Balance', balance);
+```
+
+This should show a positive balance. If no tokens appear, try to request tokens from the faucet again. If that still doesn't work, please come over to [our Discord](https://discord.iota.org/) and we'll sort it out.
+
+## Send tokens
+
+Now that we have some tokens, we can send them around.
+
+```js
+await account.sync();
+
+// To sign a transaction we need to unlock stronghold.
+await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
+
+// Replace with the address of your choice!
+const address =
+  'rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu';
+const amount = BigInt(1);
+
+const transaction = await account.send(amount, address, {
+  allowMicroAmount: true,
+});
+
+console.log(`Transaction sent: ${transaction.transactionId}`);
+
+const blockId = await account.retryTransactionUntilIncluded(
+  transaction.transactionId,
+);
+
+console.log(`Block sent: ${process.env.EXPLORER_URL}/block/${blockId}`);
+```
+
+This could take some time. The manager will automatically go through your addresses to find enough tokens to match the amount you want to send. Then it will sign the resulting transaction and send it to the node. It will warn you when you don't have enough balance, but otherwise it will show you the transaction ID, which you can use to find your transaction in the [testnet explorer](https://explorer.testnet.shimmer.network/testnet).
+
+Congratulations, you are now able to manage your tokens programmatically!
+
+## What's next?
+
+You now can create an account, generate addresses and transfer tokens. Check out the [documentation](/iota-sdk/welcome) to see what more you can do. Create some native tokens, non-fungible tokens (NFTs) and develop your own application! Have fun and good luck!
```

Comment:
This won't work as the tutorial also exist in the IOTA section where no iota-idk exists (yet :trollface: )

So for now use an absolute link: 
```suggestion
You now can create an account, generate addresses and transfer tokens. Check out the [documentation](https://wiki.iota.org/shimmer/iota-sdk/welcome) to see what more you can do. Create some native tokens, non-fungible tokens (NFTs) and develop your own application! Have fun and good luck!
```